test/unit/vl-footer.test.html

```diff
@@ -72,8 +72,44 @@
           assert.isTrue(mock.called(devUrl));
         });
       });
+
+      test('De footer stuurt een ready event als de AIV-Webuniversum-footer aan de pagina is toegevoegd', (done) => {
+        const height = 100;
```

Comment:
Height is irrelevant voor de test, dus kan verwijderd worden.

Reply:
Ik controleerde ook op hoogte omdat bij een vorige versie ook een ready-event werd verstuurt, maar op dat moment de code die uitgevoerd werd om de footer te renderen nog niet uitgevoerd was.... en de footer op dat moment dus ook nog geen hoogte had. 
Als iemand nu terug een vorige versie van vl-footer terugzet, zal de test 'de footer stuurt een ready event als de AIV-Webuniversum-footer aan de pagina is toegevoegd' groen zijn... terwijl het niet zeker is dat de footer reeds gerenderd is.

Reply:
Als ik de code bekijk, vermoed ik dat we met deze 2 testen voldoende vangnet hebben en een rode test zouden bekomen wanneer het ready event onmiddellijk verstuurd zou worden zonder te controleren dat er een footer element aanwezig is. In de source code wordt er geen rekening gehouden met de hoogte, waardoor dit geen invloed kan hebben op de testen:
- de footer stuurt een ready event als de AIV-Webuniversum-footer aan de pagina is toegevoegd
- de footer stuurt geen ready event als de AIV-Webuniversum-footer niet aan de pagina toegevoegd is

Reply:
Ja, dat klopt... nu weten we dat het 'footer' element er niet onmiddellijk is. In een eerdere versie wisten we dat nog niet.